src/processing_pipeline/stage_1.py

```
@@ -257,14 +255,30 @@ def process_audio_file(supabase_client, audio_file, local_file):
 
         print(f"Processing completed for {local_file}")
-        set_audio_file_status_success(supabase_client, audio_file["id"])
+        set_audio_file_status(supabase_client, audio_file["id"], ProcessingStatus.PROCESSED)
 
     except Exception as e:
         print(f"Failed to process audio file {local_file}: {e}")
-        set_audio_file_status_error(supabase_client, audio_file["id"], str(e))
+        set_audio_file_status(supabase_client, audio_file["id"], ProcessingStatus.ERROR, str(e))
+
+
+def reset_audio_file_status_hook(flow: Flow, flow_run: FlowRun, state: State):
```

Ruff (0.14.3)

Unused function argument: `flow`

(ARG001)